bench/many_deep.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head></head>
+
+<body>
+  <script type="module">
+    import { hydrateShadowRoots as manualWalkHydration } from '../_implementation/manual_walk.js';
+    import { hydrateShadowRoots as querySelectorHydration } from '../_implementation/queryselectorall.js';
+
+    function benchmark(hydrateMethod) {
```

Comment:
Let's add comments describing the benchmark and what's it's trying to stress.

Reply:
Done